From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aditya Prayoga <aditya@kobol.io>
Date: Sun, 13 Oct 2019 20:47:18 +0700
Subject: [ARCHEOLOGY] Added overlay support

> X-Git-Archeology: > recovered message: > U-Boot already support device tree overlay. However, it requires the
> X-Git-Archeology: > recovered message: > base dtb to be compiled with __symbol__ support.
> X-Git-Archeology: > recovered message: > This commit added patches to compile dtb with required parameter.
> X-Git-Archeology: > recovered message: > Signed-off-by: Aditya Prayoga <aditya@kobol.io>
> X-Git-Archeology: - Revision 26dcccb705cfffff1a28f3c4fb53fb950afc683c: https://github.com/armbian/build/commit/26dcccb705cfffff1a28f3c4fb53fb950afc683c
> X-Git-Archeology:   Date: Sun, 13 Oct 2019 20:47:18 +0700
> X-Git-Archeology:   From: Aditya Prayoga <aditya@kobol.io>
> X-Git-Archeology:   Subject: Added overlay support
> X-Git-Archeology:
> X-Git-Archeology: - Revision 150ac0c2afa147d9e3b036c8ecd8238fe5648cf3: https://github.com/armbian/build/commit/150ac0c2afa147d9e3b036c8ecd8238fe5648cf3
> X-Git-Archeology:   Date: Tue, 19 Nov 2019 23:25:39 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Remove K<4, change branches, new features (#1586)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2fe5e6f7553174c2b2be69d70398145e11af00b6: https://github.com/armbian/build/commit/2fe5e6f7553174c2b2be69d70398145e11af00b6
> X-Git-Archeology:   Date: Thu, 03 Dec 2020 10:24:27 +0100
> X-Git-Archeology:   From: Jannis <52237708+heisath@users.noreply.github.com>
> X-Git-Archeology:   Subject: [AR-558] Switch mvebu current to LK5.9 (based on previous mvebu-dev) (#2405)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision b619876210d23cc34652b4b4fa15cf7f61703079: https://github.com/armbian/build/commit/b619876210d23cc34652b4b4fa15cf7f61703079
> X-Git-Archeology:   Date: Wed, 04 Jan 2023 18:37:42 +0100
> X-Git-Archeology:   From: Jannis <52237708+Heisath@users.noreply.github.com>
> X-Git-Archeology:   Subject: [AR-1313] Move mvebu edge to 6.1 (#4652)
> X-Git-Archeology:
> X-Git-Archeology: - Revision ff8412d676dc175edaf07f3cd64e47b428f1c3b6: https://github.com/armbian/build/commit/ff8412d676dc175edaf07f3cd64e47b428f1c3b6
> X-Git-Archeology:   Date: Mon, 05 Aug 2024 09:38:22 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor@armbian.com>
> X-Git-Archeology:   Subject: Add patches to 6.10 folder
> X-Git-Archeology:
---
 scripts/Makefile.lib | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/scripts/Makefile.lib b/scripts/Makefile.lib
index 111111111111..222222222222 100644
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -356,6 +356,9 @@ DTC ?= $(objtree)/scripts/dtc/dtc
 DTC_FLAGS += \
 	-Wno-unique_unit_address
 
+# Enable overlay support
+DTC_FLAGS += -@
+
 # Disable noisy checks by default
 ifeq ($(findstring 1,$(KBUILD_EXTRA_WARN)),)
 DTC_FLAGS += -Wno-unit_address_vs_reg \
-- 
Armbian

